Automattic/wp-calypso
# Add Black Friday 2024 card to the My Home screen

Related to #

## Proposed Changes

This adds a task cards for the Black Friday 2024 promo.
* the card  will be visible in the primary slider for the Post Launch view

More info: pcNC1U-1u6-p2#comment-1772


## Testing Instructions



* Sandbox `public-api.wordpress.com`
* Apply D166880-code and follow instructions
* In `client/my-sites/customer-home/cards/tasks/black-friday-2024/index.jsx` append the coupon code from `36568-pb` to the `actionUrl`

* Go to `/home/{SITE_ID}?view=VIEW_POST_LAUNCH`
* You should see the Black Friday 2024 slide in the primary slider 


<img width="460" alt="Screenshot 2024-11-26 at 08 41 51" src="https://github.com/user-attachments/assets/2cb1978a-719e-41b9-954d-58c5aafe37f9">


## Pre-merge Checklist
